libr/syscall/d/meson.build

```diff
@@ -21,29 +21,43 @@ sdb_files = [
   'windows-x86-64'
 ]
 
-sdbtmp_gen_path = '@0@/@1@'.format(meson.current_source_dir(), 'gen.sh')
-sdbtmp_gen_cmd = [
+sdb_pre_script = '''#script
+import re
+import sys
+inf = open(sys.argv[1])
+outf = open(sys.argv[2], 'w')
+for line in inf:
+    if not line.startswith('_') and '=' in line:
+        arr = re.split('=|,', line)
+        print('%s.%s=%s' % (arr[1], arr[2], arr[0]), file=outf)
```

Comment:
can you use the same style for everything? either all `print(.., file=outf)` or all `outf.write(...)`?

Reply:
Fixed